data/files/datasets/src_hbase/load.hive.sql

```diff
@@ -1,3 +1,5 @@
+DROP table if exists src_hbase_tmp;
```

Comment:
Seems reasonable but why we didn't do it before and we need to it now?

Reply:
Yeps, I am not sure, but my test failed once and in the next iteration when I fixed it, It started giving me AlreadyExist exception for the table.
So, I added this. It doesn't occur on successive run when the test passes. I thought better to have this. I can try removing it, If you feel so

Reply:
No need, I was just wondering if it was somehow related to upgrade.